### PR TITLE
show activity dataviz on profiles

### PR DESCRIPTION
**Pull request by owocki**
##### Description

show activity data viz on profiles to allow easy understanding of how active a user is.

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/4908

##### Testing

see below:

Screen Shot 2019-08-29 at 3 15 51 PM
Screen Shot 2019-08-29 at 3 15 36 PM
Screen Shot 2019-08-29 at 3 15 14 PM
Screen Shot 2019-08-29 at 3 15 01 PM
Screen Shot 2019-08-29 at 3 14 43 PM
Screen Shot 2019-08-29 at 3 14 39 PM

### CODE REVIEW

File: app/dashboard/templates/profiles/heatmap.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<script type="text/javascript" src="//d3js.org/d3.v3.min.js"></script>
```
Comment (owocki) [inline app/dashboard/templates/profiles/heatmap.html]: TODO host locally

File: app/dashboard/templates/profiles/heatmap.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<script type="text/javascript" src="//cdn.jsdelivr.net/cal-heatmap/3.3.10/cal-heatmap.min.js"></script>
+<link rel="stylesheet" href="//cdn.jsdelivr.net/cal-heatmap/3.3.10/cal-heatmap.css" />
+
+<div style="color: #666; font-size: 1.2rem; padding: 0.1rem 0.7rem; border-bottom: 3px solid #FFCE08; margin-bottom: 5px;">
+<b>High</b> Activity Level; <b>Recently</b> Active
```
Comment (owocki) [inline app/dashboard/templates/profiles/heatmap.html]: TODO make dynamic

**Comment by danlipert** [comment]
Looks pretty cool - what do the white spaces represent in the pixel chart?

File: app/dashboard/templates/profiles/activity_stats.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  longest activity streak: <b>x days</b>
+  </li>
+  <li>
+  activity level: <b>high</b>
+  </li>
+  <li>
+  top skills: <b>python, django, javascript</b>
+  </li>
+  <li>
+  success pct: <b>100%</b>
```
Comment (danlipert) [inline app/dashboard/templates/profiles/activity_stats.html]: I just added a helper function for this `profile.success_rate` in https://github.com/gitcoinco/web/pull/5104

Comment (owocki) [reply app/dashboard/templates/profiles/activity_stats.html]: thanks, this will be super helpful

**Comment by owocki** [comment]
> Looks pretty cool - what do the white spaces represent in the pixel chart?

theres a legend on the bottom; darker means more engagement, ilghter means less engagement

**Pull request merged**